avocado-setup.py

```diff
@@ -393,7 +403,13 @@ def env_clean():
         helper.runcmd(cmd, ignore_status=True,
                       err_str="Error in removing package: %s" % package,
                       debug_str="Uninstalling %s" % package)
+    if os.path.isdir(prescript_dir):
```

Comment:
we should only remove the the scripts we place in, handle the cases where there is a script folder already in avocado and that has some scripts in it, not placed by wrapper, this will remove all of them...

Reply:
Actually a valid case.
We could store the list of the files we place, and remove only them.
But what happens where there is already a file existing in the same name ?

Reply:
yes, that could be another one handle, I suggest to overwrite with our/new script and log a warning to the user, optionally we can take a backup of that script, but that can be a enhancement, not the scope of this PR.

Reply:
Agreed, that is the best that can be done.